homework/style.css

```diff
@@ -1,3 +1,114 @@
+body {
+  margin-top: 0;
+  background-color: #f3f3f3;
+  width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.contribution-count {
+  font-size: 12px;
+  padding: 2px 8px;
+  line-height: 1rem;
+  background-color: gray;
+  color: #f3f3f3;
+  border-radius: 5px;
+}
+
+
+
 .alert-error {
-  color: red;
+  color: #f64153;
+  padding: 0.75rem 1.75rem;
+  margin-bottom: 1rem;
+  border-radius: 0, 25rem;
+  background-color: #f8cdd1;
+}
+
+.header-class {
+  background-color: rgb(36, 36, 151);
+  color: aliceblue;
+  font-size: 20px;
+  padding: 10px 20px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+}
+
+.repos-container,
+.contributor-container {
+  margin-bottom: 10px;
+  border: none;
+  border-radius: 5px;
+  background-color: white;
+  box-shadow: 0 1px 8px 0 rgba(0, 0, 0, 0.2), 0 3px 4px 0 rgba(0, 0, 0, 0.14), 0 3px 3px -2px rgba(0, 0, 0, 0.12);
+  padding: 10px;
+  width: 47%;
+}
+
+.select-class {
+  margin-left: 20px;
+  width: 250px;
+  height: 40px;
+  padding: 3px;
+  font-size: 14px;
+
+}
+
+th {
+  width: 80px;
+  line-height: 25px;
+}
+
+td {
+  vertical-align: top;
+}
+
+td:first-child {
+  width: 100px;
+  min-width: 100px;
+  max-width: 100px;
+}
+
+.avatar {
+  border-radius: 5px;
+  margin-right: 15;
+  width: 100px;
+}
+
+.main-container {
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding-top: 10px;
+}
+
+a {
```

Comment:
This CSS rule should only apply to the `<a>` in the contributor's section. It screws up the `<a>` in the repo table. Therefore you should restrict to children of the `contributor-details` class:

```css
.contributor-details a {
  ...
}
```

Reply:
This is also done.
Thank you for your eagle eye.